compiler-cli/src/http.rs

```diff
@@ -42,3 +46,31 @@ impl gleam_core::io::HttpClient for HttpClient {
             .map_err(Error::http)
     }
 }
+
+fn init_client() -> Result<&'static Client, Error> {
+    if let Some(client) = REQWEST_CLIENT.get() {
+        return Ok(client);
+    }
+
+    let certificate_path = std::env::var("GLEAM_CACERTS_PATH").map_err(|error| Error::FileIo {
```

Comment:
This is reading an environment variable, not reading a directory.

I think this is also going to error and refuse to compile code unless the `GLEAM_CACERTS_PATH` variable is set?